apps/studio/data/subscriptions/project-addon-update-mutation.ts

```diff
@@ -42,11 +43,12 @@ type ProjectAddonUpdateData = Awaited<ReturnType<typeof updateSubscriptionAddon>
 export const useProjectAddonUpdateMutation = ({
   onSuccess,
   onError,
+  suppressToast,
```

Comment:
we don't need an additional suppressToast property here - we just need to pass an onError handler to where we call this query, it'll override the default onError behaviour here

Reply:
in this scenario. we are using mutateAsync, and catching the error with await.
that still work for this?